CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 #### :rocket: New Feature
 - Variants: Allow coercing from variant to variant, where applicable. https://github.com/rescript-lang/rescript-compiler/pull/6314
 
+#### :boom: Breaking Change
+- Fixed the issue of name collision between the newly defined Js.Json.t and the variant constructor in the existing Js.Json.kind type. To address this, the usage of the existing Js.Json.kind type can be updated to Js.Json.Kind.s. https://github.com/rescript-lang/rescript-compiler/pull/6317
```

Comment:
s / s / t

Reply:
s / kind / t https://github.com/rescript-lang/rescript-compiler/pull/6317/commits/7f3959d556c6ccd86916fc8030e8b05827afa90d

Reply:
I thought it's idiomatic for types

Reply:
It ended up being `t`, `s` was just a confusion.

Reply:
It is t.
```
type t

module Kind = {
  type json = t
  type t
}
```

Reply:
I see 😁